jest-dom/__tests__/jest_dom_test.ml

```diff
@@ -1,3 +1,11 @@
+[@@@mel.config { flags = [| "--preamble";
+{|
+/**
+ * @jest-environment jsdom
```

Comment:
Should this be added in `jest.config.js`? Or is it beneficial to keep it here for more accuracy?